postman/template-env.secure-file.json.txt

```diff
@@ -38,7 +38,7 @@ ad ambiente usando la seguente notazione <NOME_PARAMETRO>, mentre per i secret <
     },
     {
       "key": "baseurlArc",
-      "value": "https://api.<ENV>.cittadini.pagopa.it/arc/v1",
+      "value": "https://citizen.internal.<ENV>.cittadini.pagopa.it/arcbe",
```

Comment:
This way you are moving all the API call towards the ingress endpoint.
Maybe it could be bettere to have 2 properties:
1. baseUrlArc
2. baseInnerUrlArc

Otherwise you will not more testing the public exposure

Reply:
added inner url and updated postman collection